playground/src/state-view/state-view.scss

```diff
@@ -0,0 +1,20 @@
+@import 'base.scss';
+@import 'components/atoms/_text.scss';
+
+.state-view {
+  padding: 25px;
+
+  &-loader {
```

Comment:
As stated earlier it could make sense to make the loader a standalone component to have it reusable.